web/src/lib/components/album-page/album-viewer.svelte

```diff
@@ -32,14 +32,57 @@
 
   dragAndDropFilesStore.subscribe((value) => {
     if (value.isDragging && value.files.length > 0) {
-      handlePromiseError(fileUploadHandler(value.files, album.id));
+      if (isExternalUser()) {
+        if (
```

Comment:
why do we need `if` here, there is no condition for this one?

Reply:
ah wait nvm, the condition is on the second line)

Reply:
Using `confirm` doesn't look that nice. We use `<ConfirmDialogue>` in a lot of places to handle similar situations, what do you think of using that?

Also having this message for every upload can get annoying and I think there should be a way to disable it or just show it once.